variables.tf

```diff
@@ -29,6 +29,12 @@ variable "create_hmac_key" {
   default     = true
 }
 
+variable "resource_key_existing_serviceid_crn" {
+  description = "CRN of existing serviceID to bind with resource key to be created"
```

Comment:
what happens if this is left at null? A new service Id is created? If so, lets say that in the description here please

Reply:
yes, if null the behavior is exactly the same of current one where no serviceID is specified